examples/webpack.config.js

```diff
@@ -45,7 +47,14 @@ const config = {
 			{
 				test: /\.(js|jsx)?$/,
 				exclude: /node_modules/,
-				loaders: ['babel-loader'],
+				loaders: [
+					{
+						loader: 'babel-loader',
+						options: {
+							...JSON.parse(fs.readFileSync(path.resolve(__dirname, './.babelrc'))),
```

Comment:
Did you find that this is not automatically using the options from the `.babelrc` file?

Webpack is still fairly outdated here, so the docs don't lineup well with this version.  It looks like there is a possibility that a property `babelrc: true` can be added in the options object to cause it to pick it up.